### PR TITLE
feat:(heroes-radar): implement additional filters

### PR DESCRIPTION
**Pull request by ThorsAngerVaNeT**
**🟢 Add `deploy` label if you want to deploy this Pull Request to staging environment**

#### 🧑‍⚖️ Pull Request Naming Convention

- Title should follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- Do not put issue id in title
- Do not put WIP in title. Use [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) functionality
- Consider to add `area:*` label(s)

* [x] I followed naming convention rules

---

#### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Test Case
- [ ] Documentation update
- [ ] Other

#### 🔗 Related issue link

Partly done #2194
- The dashboard should allow filtering by country (**available only for admins**) and time period.
- The badge list should have a hover feature showing the provided text and date
- The dashboard should be accessible to all RS APP users.

Also add exporting to csv file

#### 💡 Background and solution

image

#### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Database migration is added or not needed
- [x] Documentation is updated/provided or not needed
- [x] Changes are tested locally

### CODE REVIEW

File: client/src/components/Heroes/HeroesRadarTab.tsx [hunk 1]
```diff
@@ -69,24 +105,62 @@ function HeroesRadarTab({ setLoading, courses }: { setLoading: (arg: boolean) =>
...
+            </Form.Item>
+          )}
+          <Form.Item name={'dates'} label="Dates" style={{ minWidth: 260, marginBottom: 16 }}>
+            <RangePicker presets={rangePresets} />
+          </Form.Item>
+          <Form.Item name={'notActivist'} valuePropName="checked" style={{ marginBottom: 16 }}>
+            <Checkbox>Show only not activists</Checkbox>
+          </Form.Item>
+          <Space align="start" size={20} style={{ marginBottom: 16 }}>
+            <Button size="middle" type="primary" htmlType="submit">
```
Comment (valerydluski) [inline client/src/components/Heroes/HeroesRadarTab.tsx:150]: middle is default value

Comment (ThorsAngerVaNeT) [reply client/src/components/Heroes/HeroesRadarTab.tsx:150]: got it, I've copy-pasted it from Gratitudes form, should I fix it there too?

Comment (ThorsAngerVaNeT) [reply client/src/components/Heroes/HeroesRadarTab.tsx]: fixed in `HeroesRadarTab.tsx`

File: client/src/api/.openapi-generator/FILES [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
-git_push.sh
-index.ts
+.gitignore
+.npmignore
+api.ts
+base.ts
+common.ts
+configuration.ts
+git_push.sh
+index.ts
```
Comment (apalchys) [inline client/src/api/.openapi-generator/FILES:8]: What changed in this file?

File: client/src/components/Heroes/HeroesRadarTab.tsx [hunk 3]
```diff
@@ -19,25 +26,53 @@ const initialPage = 1;
 const initialPageSize = 20;
 const initialQueryParams = { current: initialPage, pageSize: initialPageSize };
 
+const { RangePicker } = DatePicker;
+
+const rangePresets: TimeRangePickerProps['presets'] = [
+  { label: 'Last 7 Days', value: [dayjs().add(-7, 'd'), dayjs()] },
```
Comment (apalchys) [inline client/src/components/Heroes/HeroesRadarTab.tsx:32]: please reuse `dayjs()` for optimization purposes

**Review by apalchys — APPROVED** [review]

File: client/src/api/.openapi-generator/FILES (hunk 2 above)
Comment (ThorsAngerVaNeT) [reply client/src/api/.openapi-generator/FILES:8]: > What changed in this file?

I believe EOL, I should revert it.
Thank you

File: client/src/components/Heroes/HeroesRadarTab.tsx (hunk 3 above)
Comment (ThorsAngerVaNeT) [reply client/src/components/Heroes/HeroesRadarTab.tsx]: fixed

File: client/src/api/.openapi-generator/FILES (hunk 2 above)
Comment (ThorsAngerVaNeT) [reply client/src/api/.openapi-generator/FILES]: reverted

**Pull request merged**